tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml

```diff
@@ -1,3 +1,11 @@
+resources:
```

Comment:
so many duplicated resource snippets, could we put them in one template

Reply:
No. Azure devops doesn't have this feature for resources.